style.css

```diff
@@ -0,0 +1,181 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-color: #ECEEF1;
+}
+
+a {
+    text-decoration: none;
+    color: #0077C5;
+}
+
+main {
+    /* width: 100%; */
+    border: 1px solid transparent;
+    display: block;
+}
+
+header {
+    float: right;
+    margin: 10px 10px 15px 50px;
+
+}
+
+.account {
+    line-height: 1.6;
+    float: right;
+    margin-top: 5px;
+    font-size: 10px;
+    /* margin-right: 5px; */
+    display: inline-flex;
+    text-align: right;
+    margin-right: 10px;
+}
+
+.acc {
+    line-height: 1.5;
+    font-size: 12px;
+    text-align: right;
+    /* margin: 0; */
+}
+
+.acc-link {
+    text-align: right;
+    line-height: 1.5;
+}
+
+.ol {
+    width: 100px;
+    border: 1px solid black;
+    text-align: center;
+    padding-top: 5px;
+    margin-left: 10px;
+    height: 25px;
+    font-size: 15px;
+    background-color: #0077C5;
+    border: 1px solid gray;
+    border-radius: 4px;
+    color: white;
+    cursor: pointer;
+
+
+}
+
+section {
+    margin: 75px 380px 20px 450px;
+    display: block;
+    /* border: 1px solid black; */
+    justify-content: center;
+    align-items: center;
+}
+
+.center {
+    display: block;
+    /* padding: 20px 20px 0 20px; */
+    width: 380px;
+    /* margin: 0 auto; */
+    border: 1px solid gray;
+    border-radius: 5px;
+    background-color: white;
+    margin-bottom: 50px;
+    color: #4C4C4C;
+}
+
+.center-top {
+    text-align: center;
+    margin-bottom: 5px;
+    font-family: Arial, Helvetica, sans-serif;
+
+
+}
+
+.center-top>h5 {
+    line-height: normal;
+}
+
+form {
+    padding: 20px;
+}
+
+.insid {
+    margin: 15px 0;
+}
+
+.insidee {
+    margin-top: 20px;
+    margin-bottom: 5px;
+}
+
+.insides {
+    font-size: 10px;
+    margin-bottom: 30px;
+    margin-top: 0;
+}
+
+.lock {
+    position: relative;
+}
+
+span {
+    position: absolute;
+    right: 15px;
+    top: 15px;
+    color: #4C4C4C;
+}
+
+/* .insides i{
+    position: relative;
+    right: 5px;
+} */
+.input-field {
+    width: 97%;
+    height: 110%;
+    padding-top: 10px;
+    /* padding-bottom: 10px; */
+    /* background-color: ; */
```

Comment:
Please remove these comments from your CSS file and any other comment that exists.